dotcom-rendering/cdk/lib/dotcom-rendering.test.ts

```diff
@@ -0,0 +1,15 @@
+import { App } from 'aws-cdk-lib';
+import { Template } from 'aws-cdk-lib/assertions';
+import { DotcomRendering } from './dotcom-rendering';
+
+describe('The DotcomRendering stack', () => {
+	it('matches the snapshot', () => {
+		const app = new App();
+		const stack = new DotcomRendering(app, 'DotcomRendering', {
+			stack: 'frontend',
+			stage: 'TEST',
+		});
+		const template = Template.fromStack(stack);
+		expect(template.toJSON()).toMatchSnapshot();
+	});
+});
```

Comment:
I think I get it - but what security does this give us?

Reply:
Makes sure that the checked in snapshot of our cloudformation config matches that which CDK generates. Essentially makes sure that we don't introduce changes to our infrastructure without intending to as the test will fail if that happens.